src/middlewares/isRequester.js

```diff
@@ -0,0 +1,18 @@
+import Customize from '../helpers/Customize';
+// eslint-disable-next-line import/named
```

Comment:
Remove this comment and resolve the eslint error.

Reply:
I have removed them